extensions/amp-analytics/0.1/vendors/gtag.js

```diff
@@ -17,6 +17,12 @@
 export const GTAG_CONFIG = /** @type {!JsonObject} */ ({
   'configRewriter': {
     'url': 'https://www.googletagmanager.com/gtag/amp',
+    'varGroups': {
+      'dns': {
+        'dr': 'DOCUMENT_REFERRER',
+        'dl': 'SOURCE_URL',
```

Comment:
Is this the group we wanted enabled by default?

Reply:
No, this one must be explicitly enabled by user